beater/config/java_attacher.go

```diff
@@ -23,8 +23,9 @@ import "fmt"
 // attacher jarfile.
 type JavaAttacherConfig struct {
 	Enabled        bool                `config:"enabled"`
-	DiscoveryRules []map[string]string `config:"discovery_rules"`
-	Config         map[string]string   `config:"config"`
+	DiscoveryRules []map[string]string `config:"discovery-rules"`
+	Config         []string            `config:"config"`
```

Comment:
Why did you change from `map[string]string` to `[]string`?
I thought we settled on this syntax in the yaml file:

```yaml
  config
    foo: bar
    baz: qux
```

That seems nicer than 
```yaml
  config
    - "foo=bar"
    - "baz=qux"
```

Reply:
you're probably correct, as that's what was there. I'm guessing i saw the example test i had from before and got confused, and backed out the wrong change